html/controls/sk/sk-sheet.js

```diff
@@ -154,48 +142,95 @@ function prepareSkSheetTable(element, teamId, mode) {
 					var current = isTrue(WS.state[prefix+'ScoringTrip('+t+').Current']);
 					row.find('.Trip'+t).text(score == null ? '' : current && score == 0 ? '.' : score);
 					otherRow.find('.Trip'+t).text('');
-				})
-			});
-			WS.Register([prefix+'ScoringTrip(10).Score', prefix+'ScoringTrip(10).AfterSP', 
-						prefix+'ScoringTrip(11).Score', prefix+'ScoringTrip(11).AfterSP', 
-						prefix+'ScoringTrip(12).Score', prefix+'ScoringTrip(12).AfterSP'], function (k, v) {
-				var scoreBeforeSP = '';
-				var scoreAfterSP = '';
-				$.each(new Array(3), function(idx) {
-					var t = idx + 10;
-					var tripScore = WS.state[prefix+'ScoringTrip('+t+').Score'];
-					if (tripScore != null) {
-						if (isTrue(WS.state[prefix+'ScoringTrip('+t+').AfterSP'])) {
-							scoreAfterSP = scoreAfterSP=='' ? tripScore : scoreAfterSP + "+" + tripScore;
-						} else {
-							scoreBeforeSP = scoreBeforeSP=='' ? tripScore : scoreBeforeSP + "+" + tripScore;
+				} else if (k.parts[4] == 'ScoringTrip' && k.ScoringTrip > 10) {
+					var scoreBeforeSP = '';
+					var scoreAfterSP = '';
+					$.each(new Array(3), function(idx) {
```

Comment:
Since you're now catching any trip index, this should be a while loop that ends, when a trip score is `null`.

Reply:
Done